extensions/wikia/UserLogin/js/UserLoginModal.js

```diff
@@ -80,51 +78,18 @@
 						window.FacebookLogin.init(window.FacebookLogin.origins.MODAL);
 					}
 
-					UserLoginModal.loginAjaxForm = new window.UserLoginAjaxForm($loginModal, {
-						ajaxLogin: true,
-						// context is this instance of UserLoginAjaxForm
-						callback: function (res) {
```

Comment:
I was able to simplify this section quite a bit by tracing back all the places that set these options and call these functions.